### PR TITLE
rewrite the README

### PR DESCRIPTION
**Pull request by technoweenie**
Assumes #9 was merged.

### CODE REVIEW

**Comment by technoweenie** [comment]
@simeonwillbanks: I took your suggestion and moved the bulk of the docs to grohl.go.  I want to go through and tidy this lib up for a 1.0 release this week.

**Comment by simeonwillbanks** [comment]
:thumbsup: 

Maybe add a quick Getting Started section to the README?

Borrowing [goworker example](https://github.com/benmanns/goworker/blob/master/README.md). (Please forgive compile errors or small mistakes :smile:)

``` go
package main

import (
    "fmt"
    "github.com/technoweenie/grohl"
    "github.com/benmanns/goworker"
)

func myFunc(queue string, args ...interface{}) error {
    fmt.Printf("From %s, %v", queue, args)
    return
}

func init() {
    goworker.Register("MyClass", myFunc)
}

func main() {
    grohl.AddContext("app", "worker")
    grohl.AddContext("deploy", os.Getenv("DEPLOY"))

    if err := goworker.Work(); err != nil {
        grohl.Log(grohl.Data{"fn": "myFunc", "error": err})
    }
}
```

File: grohl.go [hunk 1]
```diff
@@ -1,3 +1,186 @@
...
+
+	statter, err := g2s.Dial("udp", "statsd.server:1234")
+	if err != nil {
+		panic(err)
+	}
+	grohl.CurrentStatter = statter
+
+Once being set up, the statter functions above will not output any logs.
+
+Scrolls makes it easy to measure the run time of a portion of code.
```
Comment (simeonwillbanks) [inline grohl.go]: _Grohl_ makes it easy to measure the run time of a portion of code.

**Comment by technoweenie** [comment]
:+1:

File: grohl.go [hunk 2]
```diff
@@ -1,3 +1,186 @@
...
+	grohl.Log(grohl.Data{"status": "exec"})
+	timer.Finish()
+
+This would output:
+
+	fn=test at=start
+	status=exec
+	fn=test at=finish elapsed=0.300
+
+You can change the time unit that Scrolls uses to "milliseconds" (the default is
```
Comment (simeonwillbanks) [inline grohl.go]: You can change the time unit that \* Grohl\* uses to "milliseconds"

**Comment by simeonwillbanks** [comment]
Nice README example!

**Comment by technoweenie** [comment]
Thanks for the feedback.  Rolling with this.  Really close to getting the first public release out.  Just want to see how these changes work on my app.

**Comment by simeonwillbanks** [comment]
Anytime and sweet.

**Pull request merged**